cdap-ui/app/features/tracker/templates/preview.html

```diff
@@ -0,0 +1,64 @@
+<!--
+  Copyright © 2016 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+<div class="content-loading text-center" ng-show="PreviewController.loading">
+  <h3>
+    <span class="fa fa-spinner fa-spin"></span>
+    <span>Obtaining preview data</span>
+  </h3>
+</div>
+
+<div class="preview-table-container" ng-hide="PreviewController.loading">
+  <table class="table table-bordered" ng-if="PreviewController.previewData.length > 0">
+    <thead>
+      <th></th>
+      <th ng-repeat="header in ::PreviewController.previewSchema">
+        {{ header.name }}
+      </th>
+    </thead>
+
+    <tbody>
+      <tr ng-repeat="row in ::PreviewController.previewData">
```

Comment:
Are we having any limits on the data count we show? We should probably use `ng-inifiniteScroll` here and start with 10. It will make things much fast (rather than loading all 500 rows if we get).